autotest.pm

```diff
@@ -22,7 +22,7 @@ use Mojo::IOLoop::ReadWriteProcess::Session 'session';
 use Mojo::File qw(path);
 use File::Glob qw(bsd_glob);
 
-our @EXPORT_OK = qw(loadtest $selected_console $last_milestone_console query_isotovideo);
+our @EXPORT_OK = qw(loadtest $selected_console $last_milestone_console query_isotovideo _openqa_require_lua);
```

Comment:
If the function is exported it should probably not start with an underscore as this is usually only used for module-internal functions.